### ISSUE TITLE
Disable hover styles for touch devices

### ISSUE THREAD

**Issue by saadeghi** [saadeghi/daisyui#1690]
### Describe your issue

use `@media (hover: hover)`

**Comment by manthanindane** [comment]
Is this issue solved?

**Comment by saadeghi** [comment]
No. It will be a breaking change. It will be solved in the next major update (v3.0)  
For now, you can manually add CSS to customize hover styles or you can use a PostCSS plugin like this: https://www.npmjs.com/package/postcss-hover-media-feature

**Comment by manthanindane** [comment]
Okay

**Comment by saadeghi** [comment]
Fixed in `3.0`
